### DISCUSSION TITLE
Support for LLMonitor or other APM-like chain tracing tools

### DISCUSSION THREAD

**Discussion by mousounikki** [FlowiseAI/Flowise#937] [Ideas]
Currently flowise only supports tracking APM call links by viewing them in console logs, which is difficult for low-code debugging of each step's output and building applications in production environments. Already seen that LLMonitor is supported in langchain.js, is it a follow up to consider adding a tracking id that integrates LLMonitor by default in every flowise app? similar to LangSmith's private deployment solution

**Additional context**
https://js.langchain.com/docs/ecosystem/integrations/llmonitor
https://github.com/llmonitor/llmonitor

**Comment by vinodkiran** [comment]
PR Submitted :: https://github.com/FlowiseAI/Flowise/pull/952
